app/models/manageiq/providers/datawarehouse_manager.rb

```diff
@@ -1,5 +1,8 @@
 module ManageIQ::Providers
   class DatawarehouseManager < BaseManager
+    has_many :datawarehouse_nodes, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :advanced_settings, :as => :resource, :dependent => :destroy
```

Comment:
@enoodle if the fields are: version, and health status data, then they seem more custom attributes than advanced settings.